elide-core/src/main/java/com/yahoo/elide/jsonapi/EntityProjectionMaker.java

```diff
@@ -307,6 +308,15 @@ public EntityProjection visitIncludePath(Path path) {
                 .map(attributeName -> Attribute.builder()
                     .name(attributeName)
                     .type(dictionary.getType(entityClass, attributeName))
+                    .arguments(
+                            dictionary.getAttributeArguments(entityClass, attributeName)
```

Comment:
JSON-API doesn't really support attribute arguments.  We'll need to augment the GraphQL logic that builds the entity projection instead.